Makefile

```diff
@@ -227,3 +227,35 @@ endif
 
 $(TBB_LIBRARIES_BUILD_LOCATIONS) $(SUNDIALS_LIBRARIES_BUILD_LOCATIONS): | build/math-$(MATH_VERSION)
 	$(MAKE) -f Makefile.libraries $@
+
+###############################################################################
+# Precompile httpstan-related objects, eventually linked in httpstan/models.py
+###############################################################################
+
+
+PYTHON_CC ?= $(shell python3 -c 'import sysconfig;print(" ".join(sysconfig.get_config_vars("CC")))')
+PYTHON_CFLAGS ?= $(shell python3 -c 'import sysconfig;print(" ".join(sysconfig.get_config_vars("CFLAGS")))')
+PYTHON_CCSHARED ?= $(shell python3 -c 'import sysconfig;print(" ".join(sysconfig.get_config_vars("CCSHARED")))')
+PYTHON_INCLUDE ?= -I$(shell python3 -c'import sysconfig;print(sysconfig.get_path("include"))')
+PYTHON_PLATINCLUDE ?= -I$(shell python3 -c'import sysconfig;print(sysconfig.get_path("platinclude"))')
```

Comment:
I don't know what the best practice here is. I thought it was to always use `python3`. This might be a Linux thing since the current version of Debian/Ubuntu does not even provide a `python` executable.